lib/net/pool.js

```diff
@@ -1883,6 +1898,121 @@ class Pool extends EventEmitter {
     }
   }
 
+  /**
+   * Handle peer `getcfilters` packet.
+   * @method
+   * @private
+   * @param {Peer} peer
+   * @param {GetCFiltersPacket} packet
+   */
+
+  async handleGetCFilters(peer, packet) {
+    if (!this.options.bip157)
+      return;
+
+    const stopHeight = await this.chain.getHeight(packet.stopHash);
+    // The stop hash doesn't exist
+    if (!stopHeight)
+      return;
+
+    if (stopHeight - packet.startHeight + 1 >= common.MAX_CFILTERS)
+      return;
+
+    const indexer = this.getFilterIndexer(filtersByVal[packet.filterType]);
+    if (!indexer)
+      return;
+
+    // console.log('pool,1928', stopHeight, packet.startHeight);
```

Comment:
I think you missed a couple of these. The other is on L1968.
What I prefer to do is `cmd+shift+F` and do a whole project search for any `console.log`